content/docs/quickstart/sqlalchemy.md

```diff
@@ -2,13 +2,23 @@
 title: Run a SQL Alchemy App
 ---
 
+### Introduction
+
 SQLAlchemy is among the most popular ORMs in the Python universe. Because Neon is fully compatible with vanilla PostgreSQL, you only need to fill in the correct connection details.
 
-Prerequisites:
+## Prerequisites
+
+To complete this section, you will need:
+
+- Already created a Project on Neon Cloud service
```

Comment:
strange phrasing, probably should be `Created project on Neon Cloud`

Reply:
Rephrased it, please see commit, https://github.com/neondatabase/website/pull/59/commits/18c9b66d62c94fa00939aad97366235a686133b2.

It should read like this,

"To complete this section, you will need to already have the following . . . created a Project on Neon Cloud Service, have SQLALchemy installed with . . . "

Let me know if it doesn't.